vivisect/tests/testpe.py

```diff
@@ -16,6 +16,23 @@ def setUpClass(cls):
         cls.vw_psexec.loadFromFile(cls.psexec_fn)
         cls.vw_psexec.analyze()
 
+    def test_xrefs_types(self):
+        vw = self.vw_psexec
+        for xfrom, xto, xtype, xflags in vw.getXrefs():
+            self.assertTrue(isinstance(xfrom, int))
+            self.assertTrue(isinstance(xto, int))
```

Comment:
so.... what are these really testing?
if there are xrefs, they're going to be ints (or longs...)
if there aren't xrefs, this will never be run.

Reply:
That's actually the point. Before we were shoving tuples in as the second member of the tuple in certain cases. We shouldn't be doing that, as all xrefs should conform to being typed as a tuple of ints *always*.

That being said, as soon as I wrote the test, it failed, so somebody else is being naughty. So I'm not done fixing things, and there's still some other tests I want to write, but I'm currently swamped for most of today with a big meeting, so I'll finish those later tonight.

Reply:
so that is real value, but i was confused by the location of the test. 
this seems like something that should be verified against *all* of our test-binaries, not just this one PE.
it's an incredibly cheap test, but wants a more central location, imho.

or at least in a couple different files, since we don't want to have to redo analysis to get to do this test.  it would stop being cheap at that point.

Reply:
That's fair. I can expand it to all of the test binaries in this file, but I'd rather not 

Ideally one day I'd like to centralize the analysis of all our test binaries and run tests against them all, but for now, I'll leave it to this file since this testfile should have a couple in it we can test.

Reply:
i like to make things like `self.assertEqual((xfrom, isinstance(xfrom)), (xfrom, True))`

so that when it fails, it also prints out what xfrom really is.

cuz right now we just know that it's failing.

Reply:
That's a fair point. I'm switching other bits of it around so I'll incorporate that :+1: